tool/teleport/common/teleport.go

```diff
@@ -953,3 +959,22 @@ func onIntegrationConfAWSOIDCIdP(params config.IntegrationConfAWSOIDCIdP) error
 
 	return nil
 }
+
+func onIntegrationConfListDatabasesIAM(params config.IntegrationConfListDatabasesIAM) error {
+	ctx := context.Background()
+
+	iamClient, err := awsoidc.NewListDatabasesIAMConfigureClient(ctx, params.Region)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	err = awsoidc.ConfigureListDatabasesIAM(ctx, iamClient, awsoidc.ConfigureIAMListDatabasesRequest{
+		Region:          params.Region,
+		IntegrationRole: params.Role,
```

Comment:
do we want to allow users to choose the policy name? It always use the default "ListDatabases" since we don't parameterize the request here

Reply:
Removed :+1: